src/detectors/svelte.js

```diff
@@ -0,0 +1,36 @@
+const {
+  hasRequiredDeps,
+  hasRequiredFiles,
+  getYarnOrNPMCommand,
+  scanScripts
+} = require("./utils/jsdetect");
+
+module.exports = function() {
+  // REQUIRED FILES
+  if (!hasRequiredFiles(["package.json"])) return false;
+  // REQUIRED DEPS
+  if (!hasRequiredDeps(["svelte"])) return false;
+
+  /** everything below now assumes that we are within vue */
```

Comment:
Pedantic copy/paste leftover in comment. :)

Reply:
heh